rust/stackablectl/src/cmds/stack.rs

```diff
@@ -303,14 +306,15 @@ fn describe_cmd(
     }
 }
 
-#[instrument(skip(cli, stack_list, transfer_client))]
+#[instrument(skip(cli, stack_list, transfer_client), fields(indicatif.pb_show = true))]
```

Comment:
Note to self, we should `skip_all` here, since args ends up looking so bad in the logs (since it uses the Debug impl). Nothing related to this PR though.